Makefile

```diff
@@ -40,6 +40,24 @@ PLUGIN_NAME=boards
 
 export GO111MODULE=on
 
+# FIPS Support - similar to mattermost server
+# To build FIPS-compliant plugin: make dist-fips
+# Requires Docker to be installed and running
+FIPS_ENABLED ?= false
+FIPS_IMAGE ?= cgr.dev/mattermost.com/glibc-openssl-fips:15.1@sha256:d4ae220f588b914bede8a46a151addba99c894a2fa54bcf0bd090503b01b9644
```

Comment:
We need to build using the https://images.chainguard.dev/directory/image/go-msft-fips image. The glibc-opelssl-fips is the runtime one in server, which we don't need here.